README.md

```diff
@@ -28,4 +73,23 @@ with tracer.start_span('foo'):
 
 ## Contributing
 
-See [CONTRIBUTING.md](CONTRIBUTING.md)
\ No newline at end of file
+See [CONTRIBUTING.md](CONTRIBUTING.md)
+
+## Release Schedule
+
+OpenTelemetry Python is under active development. Our goal is to release an
+_alpha_ version of the library at the end of September 2019. This release isn't
+guaranteed to conform to a specific version of the specification, and is likely
+to include backwards-incompatible changes in later releases.
+
+| Component                   | Version | Target Date       |
+| --------------------------- | ------- | ----------------- |
+| Tracing API                 | Alpha   | September 30 2019 |
+| Tracing SDK                 | Alpha   | September 30 2019 |
+| Metrics API                 | Alpha   | September 30 2019 |
+| Metrics SDK                 | Alpha   | September 30 2019 |
+| Jaeger Trace Exporter       | Alpha   | Unknown           |
+| Prometheus Metrics Exporter | Alpha   | Unknown           |
+| Context Propagation         | Alpha   | September 30 2019 |
+| OpenTracing Bridge          | Alpha   | Unknown           |
```

Comment:
IIRC OpenTracing Bridge for Python has targeted 9/30, please help to double check.

Reply:
In the [Python SIG meeting of 2019-08-29](https://docs.google.com/document/d/1CIMGoIOZ-c3-igzbd6_Pnxx1SjAkjwqoYSUWxPY8XIs/edit#heading=h.rsk1uqixrfg0) with @tedsuo we said that we can't promise anything for the bridges yet because they requirements for them are underspecified. I think the remaining goals are ambitious enough (considering e.g. the current state of the context propagation spec discussion).

Reply:
I don't know that we need the OpenTracing bridge for the alpha release, and it's unlikely that we'd get the OC bridge done by then. If it looks like we'll get it out in time we can update the date, here or in another PR.